### ISSUE TITLE
[Bug]: Policy method name recognition

### ISSUE THREAD

**Issue by ProgrammerZ** [laravel-idea/plugin#912]
### Bug description

When I Ctrl + click on `update` in `Gate::authorize('update', $model); ` it is not linked to update method on Policy. Same issue is for `create`.
    

### Plugin version

8.0.2.233

### Operating system

Windows

### Steps to reproduce

  For example when I have App\Http\Controllers\AccountController

    public function show(Account $account)
    {
        Gate::authorize('view', $account);
    }

    public function update(Request $request, Account $account)
    {
        Gate::authorize('update', $account);
    }

         
And App\Policies\AccountPolicy
   
    public function view(User $user, Account $account): bool
    {
        return true;
    }
    public function update(User $user, Account $account): bool
    {
        return true;
    }

`view` links correctly to Policy `view `method correctly, but `update` links to:
   
 ```
// Illuminate\Tests\Auth\AuthAccessGateTest
public function testPoliciesAlwaysOverrideClosuresWithSameName()
{
    ...
    $gate->define('update', function () {
        $this->fail();
    });
    ....
}
```

Currently the ones that are not correctly linked: `create`, `update`. The rest are working.

**Comment by adelf** [comment]
Hello. I've tried to reproduce this but without success. Everything works flawlessly. Maybe you have another Account class?

**Comment by ProgrammerZ** [comment]
@adelf I have another project almost bare bone Laravel v10.41 installation at this point. Tried to create similar for testing: Model, Controller, Policy - same result.
It references `$gate->define`, `->gate()->define()` in `AuthAccessGateTest`, `AuthorizeMiddlewareTest`, `ValidationRuleCanTest `and probably more. So if there is a same gate name in Policy as in these test files, it will reference method in test file. 

![ide-2](https://github.com/laravel-idea/plugin/assets/5883291/205f6cfd-b79b-4282-a886-cdd64ad144ad) 

May be there is some kind of PHPStorm settings on my side, no sure.

**Comment by adelf** [comment]
Got it now. Yes, gates have a priority here. I've changed it, so for this case, it will suggest a gate and policy method for the "create" case. Two GoTo targets.

BTW, you can use the `--prefer-dist` option during `composer install`. It might help not to download tests from packages.

**Comment by adelf** [comment]
Forgot to say thank you for reporting! Really helpful!

**Comment by ProgrammerZ** [comment]
@adelf Thank you for an awesome plugin and `--prefer-dist` advise.

**Comment by adelf** [comment]
Fixed in 8.0.4. Thank you.
